R/rxsolve.R

```diff
@@ -1320,15 +1404,6 @@ rxSolve.default <- function(object, params = NULL, events = NULL, inits = NULL,
 
     }
   }
-  .n1 <- setdiff(intersect(tolower(names(params)), tolower(names(.ctl$iCov))), "id")
```

Comment:
This is no longer included (and should be)....

Reply:
Nevermind I see it now.

Reply:
Yeah, I also tried to simplify the overall dataset generation functions.